### PR TITLE
Feat : Add Card Component

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
Add Card Component with  Text only Card , Card with Shadow , Vertical card , card with text overlay , card with dismiss and card with badge

### CODE REVIEW

File: Components/card.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
+.card {
+  width: 300px;
```
Comment (Vishruta-Patil) [inline Components/card.css]: use rem instead of px.

**Review by Vishruta-Patil — CHANGES_REQUESTED** [review]
use rem instead of px. It will help in responsiveness. 
Everything else looks good.

File: Components/card.css (hunk 1 above)
Comment (Omkar-Chaskar) [reply Components/card.css]: Thanks @Vishruta-Patil for review I will do it in next PR .

**Review by Bhakti199 — APPROVED** [review]
Everything looks cool, good to merge.

**Review by Vishruta-Patil — APPROVED** [review]

**Pull request merged**